core/github_handler.py

```diff
@@ -0,0 +1,94 @@
+import logging
+
+from config import settings
+from core.base_handler import BaseHandler
+from generators.github import GithubIssueGenerator
+from targets.github import Github
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class GithubHandler(BaseHandler):
+    def issue_handler(self):
+        if not self.entities:
+            logger.info("No entities found")
+            return
+
+        logger.info("Searching for Github issues to create / update")
+
+        for entity in self.entities:
+            entity_scorecard = entity.get("scorecards", {}).get(
+                self.scorecard.get("identifier"), {}
+            )
+            rules_by_level = {"Gold": [], "Silver": [], "Bronze": []}
+
```

Comment:
there is some repeating code, between jira and github, especially when talking about the next phase of custom scorecards.